libs/agency/feature/dashboard/src/lib/components/notes/note-form/note-form.component.ts

```diff
@@ -62,6 +70,19 @@ export class NoteFormComponent {
   public mode = NoteFormMode.Add;
   public loading = true;
 
+  public actions: ActiveTaskAction[] = [
```

Comment:
Since this isn't api driven it would probably be better using the type defined in the footer actions component.
LMK what you think

```suggestion
  public actions: FooterAction[] = [
```